ads/js/utils.js

```diff
@@ -31,6 +31,23 @@ function getLayoutName () {
 	return layout;
 }
 
+function getVariantNames () {
+	let abState = '-';
+	const abStateEl = document.querySelector('[data-ab-state]');
+	if(abStateEl) {
+		abState = abStateEl.getAttribute('data-ab-state');
+	}
+	return abState;
+}
+
+let consolidateMetrics = function (container, metrics){
+	let slotName = container.dataset['oAdsName'];
```

Comment:
you're right, this can just be a function definition, rather than assign it to a var. i'll update